templates/definition/meter/openems.yaml

```diff
@@ -0,0 +1,42 @@
+template: openems
+products:
+- brand: OpenEMS
+- brand: FENECON
+guidedsetup:
+  enable: true
+params:
+- name: usage
+  choice: [ "grid", "pv", "battery" ]
+- name: host
+  required: true
+  example: 192.168.100.100
```

Comment:
Können wir hier 192.0.2.2 als "unroutable" Adresse nehmen?

Reply:
Gerne. `192.168.100.100` ist die Fallback Statische IP von FEMS, wird in der Praxis aber wohl nicht genutzt.